Plugins/BuildNotifications.PluginInterfaces/Builds/BuildReason.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace BuildNotifications.PluginInterfaces.Builds
+{
+    /// <summary>
+    /// Describes the reason why a build has been created.
+    /// </summary>
+    public enum BuildReason
```

Comment:
What is the difference between Unknown and Other?

Reply:
Other is for reasons plugin may provide that are not supported by BN. Unknown is for the case that plugins cannot provide a reason, or the reason is literally unknown.

If you have a look at [MSDN](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ff736208(v%3Dvs.120))
you see a couple of examples of build reasons we do not support/need.